scripts/docs.sh

```diff
@@ -31,3 +31,4 @@ cd docs
 pip3 install -r requirements.txt
 sphinx-build -nW -b html -d _build/doctrees . _build/html
 cd ..
+scripts/update_robots.sh
```

Comment:
[`b_docs` job](https://app.circleci.com/pipelines/github/ethereum/solidity/15449/workflows/9592e936-c74a-4d76-b533-5cd7be2d9865/jobs/701776) is failing on this:
```
Warning, treated as error:
html_extra_path entry '/root/project/docs/_static/robots.txt' does not exist
```

I think it's because `sphinx-build` expects `robots.txt` to be already present but it does not exist until your script runs. Moving the script above the `sphinx-build` or `pip3` command should fix it.